pkg/attestation/environment/environment.go

```diff
@@ -72,15 +90,25 @@ func (a *Attestor) Attest(ctx *attestation.AttestationContext) error {
 
 	variables := os.Environ()
 	for _, v := range variables {
-		parts := strings.SplitN(v, "=", 2)
-		key := parts[0]
-		val := ""
-		if len(parts) > 1 {
-			val = parts[1]
+		key, val := splitVariable(v)
+		if _, inBlockList := a.blockList[key]; inBlockList {
+			continue
```

Comment:
do we want to hash the blocked variables?

Reply:
> variables are also exposed when --trace=true. Should we remove those for now?

the command run attestor probably should have been using an embedded environment attestor so it would get this benefit without maintaining two sections of code.

i'll take a look and see if it can be arranged that way

Reply:
i'm indifferent on whether we should hash secrets or even record that secrets were set.  i can see the benefit so that it is possible to tell that the secret was set to an expected value later on without exposing the secret, but can also see the side where some may be hesitant to even expose that some secret was set in the first place.

let me think on this one.